src/EnergyPlus/ManageElectricPower.cc

```diff
@@ -2457,7 +2463,7 @@ namespace ManageElectricPower {
 		// Flow
 
 		ElecProducedCoGen = GetInstantMeterValue( ElecProducedCoGenIndex, 2 ); //whole building
-		ElecProducedFacility = ElecProducedCoGen + WholeBldgElectSummary.ElecProducedPVRate * TimeStepSys * SecInHour + WholeBldgElectSummary.ElecProducedWTRate * TimeStepSys * SecInHour; //whole building
+		ElecProducedFacility = ElecProducedCoGen + ( WholeBldgElectSummary.ElecProducedPVRate + WholeBldgElectSummary.ElecProducedWTRate + WholeBldgElectSummary.ElecProducedStorageRate ) * TimeStepSys * SecInHour; //whole building
```

Comment:
The storage used to be part of ElecProducedCoGen because it was on Cogeneration:ElectricityProduced.  Not that it's on its own end use, it needs to be included here.  That's assuming ElecProducedFacility was correct before this change.  Ultimately, ElecProducedCoGen shouldn't be handled differently here, it should be in the WholeBldgElectSummary with a new variable ElecProducedCogenRate, but I wanted to keep the changes here focused just on the storage.